.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -14,25 +14,21 @@ assignees: amarpad, ekowtaylor, talkhasib
 - **Affected Subcomponent:** Orchestrator pod/service or AGW service
 - **Deployment Environment:** e.g. AWS (orc8r), local docker-compose (orc8r), bare-metal (AGW), Vagrant (AGW)
 
-### Describe the Feature
+### Describe the problem
 
-**Is your feature request related to a known issue? Please describe.**
-A clear and concise description of what the problem is and provide links to any relevant issues
+**Description**
+A clear and concise description of what is the problem, under which scenarios or how frequent it happen
```

Comment:
@wallyrb small nit, should it read 'what is the problem' or 'what the problem is'?

Reply:
> @wallyrb small nit, should it read 'what is the problem' or 'what the problem is'?

@ekowtaylor thanks, you are right is an indirect question. I changed. Let me know if you have any other comments